example/src/main/java/com/ncorti/slidetoact/example/SampleActivity.java

```diff
@@ -65,6 +65,36 @@ protected void onCreate(Bundle savedInstanceState) {
             case R.id.button_custom_icon:
                 setContentView(R.layout.content_custom_icon);
                 break;
+            case R.id.button_custom_icon_runtime:
```

Comment:
Can I ask you to have everything inside a single `custom_icon` button?
Maybe we can have a section below the XML approach with the UI you proposed 👍

Reply:
Perhaps we don't need to have two sliders in the custom icon example. Checkout my latest commit, I'm using the same slider to demonstrate both methods of setting custom icons. Let me know what you think.

I've removed the "custom icon at runtime" views and references.